packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -1101,6 +1113,9 @@ class _AdbLogReader extends DeviceLogReader {
   // adb logcat. We are currently passing 'time', which has the format:
   // mm-dd hh:mm:ss.milliseconds Priority/Tag( PID): ....
   void _onLine(String line) {
+    if (_linesController.isClosed) {
```

Comment:
trivial nit: comment about adb exited earlier and it might be still streaming logs.

Reply:
Done